src/mame/mame.lst

```diff
@@ -1,4 +1,4 @@
-// license:BSD-3-Clause
+﻿// license:BSD-3-Clause
```

Comment:
What have you done to this line?  Ensure your editor isn't adding a zero-width no-break space to the file.